components/automate-gateway/api/iam/v2beta/request/tokens.proto

```diff
@@ -3,10 +3,12 @@ syntax = "proto3";
 package chef.automate.api.iam.v2beta;
 option go_package = "github.com/chef/automate/components/automate-gateway/api/iam/v2beta/request";
 
+import "google/protobuf/wrappers.proto";
+
 message CreateTokenReq {
     string id = 1;
     string name = 2;
-    bool active = 3;
+    google.protobuf.BoolValue active = 3;
```

Comment:
Reviewer note: in general, it's bad to change existing protobuf values. In this case, I think we should because 1) it doesn't actually change the type for the external HTTP API because BoolValue accepts JSON true and false and 2) we are in beta still for these APIs so changing them is ok for now.